### PR TITLE
new URL for CenOS Stream 8 koji

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #1045

### CODE REVIEW

**Review by Conan-Kudo — APPROVED** [review]

**Comment by xsuchy** [comment]
Do not merge until this URL is actually working - hopefully in a day or two.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl [hunk 1]
```diff
@@ -33,7 +33,7 @@ user_agent={{ user_agent }}
 [local]
 {% endif %}
 name=CentOS Stream $releasever - Koji Local - BUILDROOT ONLY!
-baseurl=https://koji.mbox.centos.org/kojifiles/repos/dist-c{{ releasever }}-stream-build/latest/$basearch/
+baseurl=https://kojihub.stream.centos.org/kojifiles/repos/dist-c{{ releasever }}-stream-build/latest/$basearch/
```
Comment (hroncok) [inline mock-core-configs/etc/mock/templates/centos-stream-8.tpl:36]:
```suggestion
baseurl=https://kojihub.stream.centos.org/kojifiles/repos/c{{ releasever }}s-build/latest/$basearch/
```

**Review by hroncok — CHANGES_REQUESTED** [review]
It works, but the URL is different.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl (hunk 1 above)
Comment (xsuchy) [reply mock-core-configs/etc/mock/templates/centos-stream-8.tpl]: Thank you.

**Comment by xsuchy** [comment]
Updated per suggestion and ready for review.

**Review by Conan-Kudo — APPROVED** [review]

**Review by praiskup — APPROVED** [review]

**Pull request merged**